### DISCUSSION TITLE
Nested inequality statements

### DISCUSSION THREAD

**Discussion by Johntron1219** [bonsai-rx/bonsai#992] [Q&A]
Hi there, 

I was wondering if there is a way to nest inequality statements (very new to Bonsai). I couldn't find anything in the forums- is there a way to build something with nodes to see if a value is greater than 0 and less than 110 and if so return True? 

Thanks!
John

**Comment by bruno-f-cruz** [comment]
Hey,

You can do it using Bonsai nodes alone (`Tuple` into `BitwiseAnd`), or using an `ExpressionTransform` with `(it > 0) and (it < 10)` in the `Expression` property.

![image](https://user-images.githubusercontent.com/7049351/182659584-336e8797-afd8-468a-a0df-b9b404876749.png)

Hope it helps!

**Reply by Johntron1219** [reply]
Great! 

Thanks so much!
